src/sweeps/bayes_search.py

```diff
@@ -375,11 +380,6 @@ def _construct_gp_data(
             # we wont use the metric, but we should pass it into our optimizer to
             # account for the fact that it is running
             current_X.append(X_norm)
-        elif run.state in [RunState.failed, RunState.crashed, RunState.killed]:
```

Comment:
how easy would it be to keep this branch but add a config to the end user for selecting between `BEST` vs `LATEST` in abnormal cases?

Reply:
done